mlflow/R/mlflow/Dockerfile.r-devel

```diff
@@ -1,4 +1,4 @@
-FROM rocker/r-ver:devel
```

Comment:
I realize that this renders `Dockerfile.r-devel` identical to `Dockerfile.dev`. In the interest of fixing CI promptly, I'd like to make a minimal change here and reevaluate whether this is the right long-term choice for R testing shortly thereafter.